webapp/loggedin/appJob/reducer.js

```diff
@@ -0,0 +1,17 @@
+import { exportReducer } from '../../utils/reduxUtils'
+
+import {
+  appJobActiveUpdate,
+  appJobStart,
+} from './actions'
+
+
+import * as AppState from '../../app/appState'
+
+const actionHandlers = {
+  [appJobStart]: (state, { job, onComplete, autoHide }) => AppState.startJob(job, onComplete, autoHide)(state),
+
+  [appJobActiveUpdate]: (state, { job }) => AppState.updateActiveJob(job)(state),
+}
+
```

Comment:
```AppState.startJob```, ```AppState.updateActiveJob``` and all state methods related to app job should be moved to a new module AppJobState